<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-A2S9SMV" failures="0" tests="8" name="com.oneapp.test_suits.EmergencyContact_testcases" time="812.708" errors="0" timestamp="2023-04-13T13:18:24 IST" skipped="0">
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_added_succesfully_testcase" time="269.484"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_deleted_succesfully_testcase" time="20.881"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_with_blank_fullName_testcase" time="60.003"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_with_blank_MobileNumber_testcase" time="41.751"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_with_blank_relationType_testcase" time="48.010"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_saving_with_no_internet_testcase" time="36.701"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_saving_with_invalid_details_testcase" time="89.542"/>
  <system-out/>
  <testcase classname="com.oneapp.test_suits.EmergencyContact_testcases" name="validate_emergency_contact_already_exist_testcase" time="246.336"/>
  <system-out/>
</testsuite> <!-- com.oneapp.test_suits.EmergencyContact_testcases -->
